client/gutenberg/extensions/simple-payments/edit.js

```diff
@@ -96,25 +98,20 @@ class SimplePaymentsEdit extends Component {
 		}
 
 		const { attributes, setAttributes } = this.props;
-		const { email, paymentId } = attributes;
-
-		this.setState( { isSavingProduct: true }, () => {
-			apiFetch( {
-				path: `/wp/v2/${ SIMPLE_PAYMENTS_PRODUCT_POST_TYPE }/${ paymentId ? paymentId : '' }`,
-				method: 'POST',
-				data: this.attributesToPost( attributes ),
-			} )
-				.then( response => {
-					const { id } = response;
-
-					if ( id ) {
-						setAttributes( { paymentId: id } );
-					}
+		const { email } = attributes;
+		const { saveEntityRecord } = dispatch( 'core' );
+
+		this.setState( { isSavingProduct: true }, async () => {
+			saveEntityRecord( 'postType', SIMPLE_PAYMENTS_PRODUCT_POST_TYPE, this.toApi() )
+				.then( record => {
+					/* eslint-disable-next-line no-console */
+					console.error( 'Saved: %o', record );
```

Comment:
This `console.error` could probably be removed before merging?

Reply:
Cleaned up in 50e064188b5bb6bfa67c3e963e0575eb24801dce